docs/source/io_formats/summary.rst

```diff
@@ -4,255 +4,131 @@
-
-    The number of lattice cells in each direction. This dataset is present only
-    when the 'type' dataset is set to 'rectangular'.
-
-**/geometry/lattices/lattice <uid>/lower_left** (*double[]*)
-
-    The coordinates of the lower-left corner of the lattice. This dataset is
-    present only when the 'type' dataset is set to 'rectangular'.
-
-**/geometry/lattices/lattice <uid>/n_rings** (*int*)
-
-    Number of radial ring positions in the xy-plane. This dataset is present
-    only when the 'type' dataset is set to 'hexagonal'.
-
-**/geometry/lattices/lattice <uid>/n_axial** (*int*)
-
-    Number of lattice positions along the z-axis. This dataset is present only
-    when the 'type' dataset is set to 'hexagonal'.
-
-**/geometry/lattices/lattice <uid>/center** (*double[]*)
-
-    Coordinates of the center of the lattice. This dataset is present only when
-    the 'type' dataset is set to 'hexagonal'.
-
-**/n_materials** (*int*)
-
-    Number of materials in the problem.
-
-**/materials/material <uid>/index** (*int*)
-
-    Index in materials array used internally in OpenMC.
-
-**/materials/material <uid>/name** (*char[]*)
-
-    Name of the material.
-
-**/materials/material <uid>/atom_density** (*double[]*)
-
-    Total atom density of the material in atom/b-cm.
-
-**/materials/material <uid>/nuclides** (*char[][]*)
-
-    Array of nuclides present in the material, e.g., 'U-235.71c'.
-
-**/materials/material <uid>/nuclide_densities** (*double[]*)
-
-    Atom density of each nuclide.
-
-**/materials/material <uid>/sab_names** (*char[][]*)
-
-    Names of S(:math:`\alpha`,:math:`\beta`) tables assigned to the material.
-
-**/tallies/tally <uid>/name** (*char[]*)
-
-    Name of the tally.
+The current version of the summary file format is 5.0.
+
+**/**
+
+:Attributes: - **filetype** (*char[]*) -- String indicating the type of file.
+             - **version** (*int[2]*) -- Major and minor version of the summary
+               file format.
+             - **openmc_version** (*int[3]*) -- Major, minor, and release
+               version number for OpenMC.
+             - **git_sha1** (*char[40]*) -- Git commit SHA-1 hash.
+             - **date_and_time** (*char[]*) -- Date and time the summary was
+               written.
+
+**/geometry/**
+
+:Attributes: - **n_cells** (*int*) -- Number of cells in the problem.
+             - **n_surfaces** (*int*) -- Number of surfaces in the problem.
+             - **n_universes** (*int*) -- Number of unique universes in the
+               problem.
+             - **n_lattices** (*int*) -- Number of lattices in the problem.
+
+**/geometry/cells/cell <uid>/**
+
+:Datasets: - **name** (*char[]*) -- Name of the cell.
+           - **universe** (*int*) -- Universe assigned to the cell. If none is
+             specified, the default universe (0) is assigned.
+           - **fill_type** (*char[]*) -- Type of fill for the cell. Can be
+             'normal', 'universe', or 'lattice'.
```

Comment:
That has been bothering me too. I think I'll change it to material as part of this PR if you're ok with it.